policy-design-lab/pdl-frontend
# #317: landing map toggle

This PR addresses issue #317. It introduces the program selection feature to the total map on the landing page. A collaborator has validated the calculated numbers displayed on the map.

## How to Test

1. Navigate to https://policydesignlab-dev.ncsa.illinois.edu/ and review the updated landing page map: <img width="1750" alt="Screenshot 2024-10-22 at 9 56 01 AM" src="https://github.com/user-attachments/assets/d609f8b5-d2b4-49d3-ad8d-2ca4eeaa2ec0">

2. You should be able to use selectors, and the map’s numbers should change based on your selection. Note that the calculation is dynamic, so the legend bar retains its uniform distribution (i.e., five equal segments), unlike the static legend bars on other landing page maps.

3. Hover your mouse over a state. You should see the full number rather than a rounded figure, ensuring users can view the total more clearly and avoid confusion from rounded values: <img width="1726" alt="Screenshot 2024-10-22 at 10 01 59 AM" src="https://github.com/user-attachments/assets/5c7b01ab-fa0d-4f0d-acd1-915c344e665c">

4. Interact with the map to verify that the calculations are correct.
